### PR TITLE
Fix lint JsDoc for amp-analytics/*.js

### PR DESCRIPTION
**Pull request by lannka**
For #15312

### CODE REVIEW

File: extensions/amp-analytics/0.1/events.js [hunk 1]
```diff
@@ -361,7 +361,11 @@ export class ClickEventTracker extends EventTracker {
     /** @private {!Observable<!Event>} */
     this.clickObservable_ = new Observable();
 
-    /** @private @const */
+    /**
+     * @private {function(!Event)}
+     * @const
+     * @param {!Event} e
```
Comment (lannka) [inline extensions/amp-analytics/0.1/events.js]: @jridgewell I had to add this param to pass the linter. however, check-types seems to not like it. what's the right way to do here?

File: extensions/amp-analytics/0.1/events.js [hunk 2]
```diff
@@ -361,10 +361,8 @@ export class ClickEventTracker extends EventTracker {
     /** @private {!Observable<!Event>} */
     this.clickObservable_ = new Observable();
 
-    /** @private @const */
-    this.boundOnClick_ = e => {
-      this.clickObservable_.fire(e);
-    };
+    /** @private {function(!Event)} @const */
```
Comment (jridgewell) [inline extensions/amp-analytics/0.1/events.js]: `@private @const {...}`

File: extensions/amp-analytics/0.1/iframe-transport.js [hunk 3]
```diff
@@ -273,23 +272,23 @@ export class IframeTransport {
...
+   * @visibleForTesting
    */
   getCreativeId() {
     return this.creativeId_;
   }
 
   /**
    * @return {string} Type attribute of parent amp-analytics instance
-   * @VisibleForTesting
+   * @visibleForTesting
```
Comment (rsimha) [inline extensions/amp-analytics/0.1/iframe-transport.js]: I'm curious if the old (mis-spelt) annotations prevented any tests from working. I'm also curious if this makes a difference in the resulting minified bundle size, now that the correct annotations are in place.

Comment (lannka) [reply extensions/amp-analytics/0.1/iframe-transport.js]: I don't think the annotation will affect test behavior. Don't know if it affect bundle size though.

Comment (rsimha) [reply extensions/amp-analytics/0.1/iframe-transport.js]: With this change: 77.19KB (https://travis-ci.org/ampproject/amphtml/jobs/385432514#L648)
Before this change: 77.2KB (https://travis-ci.org/ampproject/amphtml/jobs/385432417#L648)

Not sure if that's a real decrease or just noise.

**Pull request merged**